src/Models/ProductModel.php

```diff
@@ -18,4 +19,31 @@ public function __construct($db)
     {
         $this->db = $db;
     }
+
+    public function addProduct(ProductEntityInterface $productEntity)
+    {
+        $array = ["sku"=>$productEntity->getSku(), "name"=>$productEntity->getName(),
+            "price"=>$productEntity->getPrice(), "stockLevel"=>$productEntity->getStockLevel()];
```

Comment:
new lines for each key? just a personal preference for readability
```suggestion
        $array = [
            "sku"=>$productEntity->getSku(), 
            "name"=>$productEntity->getName(),
            "price"=>$productEntity->getPrice(),
            "stockLevel"=>$productEntity->getStockLevel()
        ];
```